README.md

```diff
@@ -18,12 +18,11 @@ Current issues are present in the [project backlog](https://github.com/orgs/inti
 ### Branch
 
 Each branch should be associated with a team member and corresponding issue created [here](https://github.com/intive/Picover/issues).
-To accomplish that, use following template `acronym/issue_number/description`, example:
-> kma/1/initial-project
+To accomplish that, use following template `issue_number/description`, example:
+> 1/initial-project
```

Comment:
Let me know what do you think

Reply:
I use alike template in my project and I can say the schema with acronym helps with finding a branch with given author.

Reply:
on the other hand, we can see the branch author in the git history as well :D i am 60 to 40 to go with the new approach :D

Reply:
@bartek977 Agree, that can be helpful but I would say that it's really uncommon, additionally taking into consideration that usually we don't have more than 5 feature branches so it's not a big deal anyway.

Reply:
@marcinWisnia I meant the acronym is helpful if you checkout on the branch using android studio plugin
![image](https://github.com/Android-Guild/Picover/assets/49306295/d8f461d6-0a49-4fc3-9f64-d2b4db292815)


I'm not blocking this change, for me both schemas are ok

Reply:
I personally like that it gives the some kind of the order in my local git:
<img width="128" alt="image" src="https://github.com/Android-Guild/Picover/assets/33199010/74822633-b872-465b-bd9e-15fa0474958b">
But it is just my personal preference so up to you guys.